core/src/BibleEngine.class.ts

```diff
@@ -961,18 +970,33 @@ export class BibleEngine {
                     childState.modifierState.title = (<IBibleContentGroup<'title'>>(
                         content
                     )).modifier = content.modifier === 'pullout' ? 'pullout' : 'inline';
-                else if (content.groupType === 'poetry') childState.modifierState.poetry = true;
+                else if (content.groupType === 'linegroup')
+                    childState.modifierState.linegroup = true;
```

Comment:
Nitpick: would prefer `lineGroup`, to follow camel case style. up to you though!

Reply:
done